src/app/BUILD.gn

```diff
@@ -340,7 +339,8 @@ static_library("app") {
     # "CommandResponseSender._h"
     # "CommandHandler._h"
     # "ReadHandler._h",
-    # "WriteHandler._h"
+    # "WriteHandler._h",
+    # "EventManagement._h"
```

Comment:
Ideally if cpp is in a target, I would expect the .h to be there as well.

Something seems off here ... do we need more splitting maybe? We should ideally not add more of these mis-matches.

Reply:
This is not needed anymore with Event Scheduler in its own file.